test/test_asin.cpp

```diff
@@ -188,6 +223,10 @@ int main()
     test_asin<decimal32_t>();
     test_asin<decimal64_t>();
 
+    test_asin_edge<decimal32_t>();
+    test_asin_edge<decimal64_t>();
+    test_asin_edge<decimal128_t>();
+
```

Comment:
nit: would it be nicer to move these lines bellow 
```
test_asin<decimal128_t>();
```

so tests remain grouped?